recipes/standardese/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "standardese" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tgz
+  sha256: 4f27dbba6d577636a622dd0dc13257e8f11ed7bab5c41808bc0943720c2cc842
+  patches:
+    - 0001-Actually-install-standardese-in-make-install.patch
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    # standardese uses semantic versioning
+    - {{ pin_subpackage("standardese") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost
```

Comment:
```suggestion
    - boost-cpp
```

Reply:
We need boost ProgramOptions and Filesystem. Afaik the headers are not enough for that.

Reply:
(or am I misunderstanding what boost-cpp is?)

Reply:
`boost` conda package is `boost-cpp` conda package + boost python libraries.

Reply:
Ah, I thought the `cpp` was referring to headers only, like `cpp` the preprocessor. Thanks for clarifying :)